modules/nlu/src/views/full/entities/ListEntity.tsx

```diff
@@ -29,22 +17,51 @@ const FuzzyTolerance = {
   Loose: 0.65
 }
 
+interface EntityState {
+  fuzzy: number
+  occurrences: NLU.EntityDefOccurrence[]
+}
+
+function EntityContentReducer(state: EntityState, action): EntityState {
+  const { type, data } = action
+  if (type === 'setStateFromEntity') {
```

Comment:
Switch/case on `type`?

Reply:
@allardy told me once that we hate switch statements ... honestly I have no problem with them

Reply:
Yeah, and I take that from Sylvain :P